charts/gateway-addons-helm/values.yaml

```diff
@@ -209,17 +207,36 @@ opentelemetry-collector:
             # Loki will convert this to k8s_pod_name label.
             value: k8s.pod.name, k8s.namespace.name
     receivers:
+      jaeger:
+        protocols:
+          grpc:
+            endpoint: "[${env:MY_POD_IP}]:14250"
+          thrift_http:
+            endpoint: "[${env:MY_POD_IP}]:14268"
+          thrift_compact:
+            endpoint: "[${env:MY_POD_IP}]:6831"
       datadog:
-        endpoint: ${env:MY_POD_IP}:8126
+        endpoint: "[${env:MY_POD_IP}]:8126"
       zipkin:
-        endpoint: ${env:MY_POD_IP}:9411
+        endpoint: "[${env:MY_POD_IP}]:9411"
       otlp:
         protocols:
           grpc:
-            endpoint: ${env:MY_POD_IP}:4317
+            endpoint: "[${env:MY_POD_IP}]:4317"
           http:
-            endpoint: ${env:MY_POD_IP}:4318
+            endpoint: "[${env:MY_POD_IP}]:4318"
+      prometheus:
+        config:
+          scrape_configs:
+          - job_name: opentelemetry-collector
+            scrape_interval: 10s
+            static_configs:
+            - targets:
+              - "[${env:MY_POD_IP}]:8888"
     service:
+      telemetry:
+        metrics:
+          address: "[${env:MY_POD_IP}]:8888"
```

Comment:
who is populating this env ?

Reply:
otel-collector pod

Reply:
we'd better change upstream charts, I do believe they should change the default value.